src/index.js

```diff
@@ -5,14 +5,14 @@ const log = require('./logger');
 
 module.exports = {
   log: log,
-  update: (keyword, limit) => {
+  update: keyword => {
     if (typeof keyword !== 'string') {
       let msg = 'Index: Keyword is required';
       log.error(msg);
-      throw Error(msg);
+      throw new Error(msg);
```

Comment:
Yeah, I'd keep new for best practices. I don't think it matters on Node, but browser have very inconsistent error traces if you user new or not.